Sources/SwiftFormat/CMakeLists.txt

```diff
@@ -0,0 +1,105 @@
+#[[
+This source file is part of the Swift System open source project
+
+Copyright (c) 2020 Apple Inc. and the Swift System project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+#]]
+
+add_library(SwiftFormat
```

Comment:
What safeguards do we have against failing to update these source listings? Will the toolchain builds start using these instead of the SPM build?

Since the CI story on this repository still isn't great (cc @shahmishal), I want to make sure that this change doesn't add undue burden on contributors. Right now, the only coverage we have is through paired PR testing in swift-syntax, which is quite slow.

Reply:
Correct, I will be putting up changes to start consuming this build for the Windows builds/